### ISSUE TITLE
Current state of the project

### ISSUE THREAD

**Issue by sebastianqdot** [Cezerin2/Server#180]
Hey there. I've got a couple of potential clients willing to create their stores and I'm trying to understand wether to go with this project or not. My main concerns are how inactive it seems to be, though the project is fairly big and functional for the amount of traction it has had.

I understand it is the rebirth of a dead project, so my question here is how many people are maintaining this and if we can expect this to be an ongoing project.

I'm willing to try this project out (I've already installed the three repos and played around a bit), and wouldn't mind helping in adding features or improving the development process, but I don't see myself making Cezerin3 if you know what I mean lol.

Long story short, let's have a chat about this project and what we can expect from it as well as discuss how we can contribute to it.

**Comment by vamcart** [comment]
Join slack chat channel cezerin2 - https://join.slack.com/t/cezerin2/shared_invite/enQtNTE5NzYxMzA5ODc5LTVkZjM4ODUwMmNlMmMyZTkxYjg5N2QxZmQ5NjA1NTg3OWM2ZjU1NzVmNWM0N2E3ZmJjM2Q3MjQ5OGFmNTBmYjg

**Comment by vamcart** [comment]
[cezerin2-config-sample.zip](https://github.com/Cezerin2/cezerin2/files/3710880/cezerin2-config-sample.zip)
